.github/actions/external_runner/main.go

```diff
@@ -50,6 +57,7 @@ func LoadConfig() Config {
 	viper.BindEnv("timeout_millis")
 	viper.BindEnv("max_log_lines")
 	viper.BindEnv("disable_logs")
+	viper.BindEnv("filter_ansible_logs")
```

Comment:
I would make `filter_ansible_logs` `true` by default so we don't output unexpected logs by mistake. WDYT?

Reply:
Moving to proposal https://github.com/newrelic/infrastructure-agent/pull/1335#discussion_r928585883

No default filters will be defined, thus no logs will be shown.